easybuild/easyconfigs/s/Shasta/Shasta-0.8.0-foss-2020b.eb

```
@@ -0,0 +1,69 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'CMakeMakeCp'
+
+name = 'Shasta'
+version = '0.8.0'
+
+homepage = 'https://github.com/chanzuckerberg/shasta'
+description = """
+The goal of the Shasta long read assembler is to rapidly produce accurate assembled sequence using DNA reads generated
+by Oxford Nanopore flow cells as input. Computational methods used by the Shasta assembler include:
+Using a run-length representation of the read sequence. This makes the assembly process more resilient to errors in
+homopolymer repeat counts, which are the most common type of errors in Oxford Nanopore reads. Using in some phases of
+the computation a representation of the read sequence based on markers, a fixed subset of short k-mers (k ≈ 10).
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/chanzuckerberg/shasta/archive/']
+sources = ['%(version)s.tar.gz']
+patches = [
+    "Shasta-%(version)s-foss-2020b-fix_links.patch"
+]
+checksums = [
+    '8e94767275fc5352ff192afed783b00b720b6214d8779edc44e9ebe636e9833e',  # 0.8.0.tar.gz
+    'dfda9a6ca33e37236cbbc27a1d65bb54d361f27ecdb2b9855e46ac6dab6e255e',  # Shasta-0.8.0-foss-2020b-fix_links.patch
+]
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('binutils', '2.35'),
+]
+
+dependencies = [
+    ('cURL', '7.72.0'),
+    ('Boost', '1.74.0'),
+    ('libpng', '1.6.37'),
+    ('SeqAn', '2.4.0'),
+    ('BLAST', '2.11.0', '-Linux_x86_64', True),
+    ('gnuplot', '5.4.1'),
+    ('Graphviz', '2.47.0', '-Java-11'),
+    ('Python', '3.8.6'),
+    ('pybind11', '2.6.0'),
+    ('Cereal', '1.3.0', '', True),
+    ('spoa', '3.4.0'),
+    ('ScaLAPACK', '2.1.0'),
+    ('OpenBLAS', '0.3.12'),
+    ('cpu_features', '0.6.0'),
+]
+
+prebuildopts = "export CPATH=$EBROOTBOOST/include/boost/:$CPATH && "
+
+buildopts = " && make install -j %(parallel)s"
+
+files_to_copy = [
+    (['shasta-install/bin/*'], 'bin'),
+    (['shasta-install/bin/*.a'], 'lib'),
+    (['shasta-install/bin/*.so'], 'lib')
```

Please use `SHLIB_EXT` constant, and you can combine these two lines into one:

```suggestion
    (['shasta-install/bin/*.a', 'shasta-install/bin/*.%s' % SHLIB_EXT], 'lib')
```

---

done